src/easylink/step.py

```diff
@@ -815,6 +815,26 @@ def set_configuration_state(
             parent_config, combined_implementations, input_data_config
         )
 
+    def _duplicate_template_step(self) -> Step:
+        """Makes a duplicate of the template ``Step``.
+
+        A duplicate of the :attr:`template_step`, i.e. the ``Step`` that may have
+        multiplicity, should contain copies of every attribute of said ``Step``
+        *except* for the :attr:`parent_step`. This is because :attr:`parent_step`
+        contains an actual ``Step`` instance and we want all of the duplicates to
+        have that same parent as opposed to copies of one.
+
+        Returns
+        -------
+            A duplicate of the :attr:`templated_step`.
+        """
+        parent_step = self.template_step.parent_step
```

Comment:
I realize this is still a draft, but am I totally off-base to say that `parent_step` is always the same as `self`? If so, can we make that explicit here?